recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/disman.te

```diff
@@ -0,0 +1,68 @@
+policy_module(disman, 0.1)
+
+########################################
+#
+# Declarations
+#
+
+type disman_t;
+type disman_exec_t;
+init_daemon_domain(disman_t, disman_exec_t)
+
+type disman_initrc_exec_t;
+init_script_file(disman_initrc_exec_t)
+
+type disman_var_run_t;
+files_pid_file(disman_var_run_t)
+init_daemon_pid_file(disman_var_run_t, file, "disman.pid")
+
+type disman_script_t;
+type disman_script_exec_t;
+corecmd_executable_file(disman_script_exec_t)
```

Comment:
Switch corecmd_exec... to:
+role system_r types disman_script_t;
+application_domain(disman_script_t, disman_script_exec_t)